backend/pkg/api/data_access/notifications.go

```diff
@@ -419,38 +417,45 @@ func (d *DataAccessService) GetValidatorDashboardNotificationDetails(ctx context
 		searchIndexSet[searchIndex] = true
 	}
 
-	result := []byte{}
-	query := `SELECT details FROM users_val_dashboards_notifications_history WHERE dashboard_id = $1 AND group_id = $2 AND epoch = $3`
-	err := d.alloyReader.GetContext(ctx, &result, query, dashboardId, groupId, epoch)
+	// -------------------------------------
+	// dashboard and group name
+	query := `SELECT
+		uvd.name AS dashboard_name,
+		uvdg.name AS group_name
+	FROM
+		users_val_dashboards uvd
+	INNER JOIN
+		users_val_dashboards_groups uvdg ON uvdg.dashboard_id = uvd.id
+	WHERE uvd.id = $1 AND uvdg.id = $2`
+	err := d.alloyReader.GetContext(ctx, &notificationDetails, query, dashboardId, groupId)
 	if err != nil {
 		if err == sql.ErrNoRows {
 			return &notificationDetails, nil
 		}
 		return nil, err
 	}
-	if len(result) == 0 {
-		return &notificationDetails, nil
+	if notificationDetails.GroupName == "" {
+		notificationDetails.GroupName = t.DefaultGroupName
 	}
-
-	buf := bytes.NewBuffer(result)
-	gz, err := gzip.NewReader(buf)
-	if err != nil {
-		return nil, err
+	if notificationDetails.DashboardName == "" {
+		notificationDetails.DashboardName = t.DefaultDashboardName
 	}
```

Comment:
**question:** Why do we set the names to default here and don't leave them as empty strings?
It should also be impossible to create a dashboard or group without a name with at least length 1.

Reply:
You're right, it's more of a debug/testing thing - some internal dashboards don't have a name so I thought it wouldn't hurt